src/main/resources/application.yml

```diff
@@ -60,13 +61,20 @@ spring:
       resourceserver:
         jwt:
           issuer-uri: http://localhost:8082/realms/mlt-local
+          audiences: http://localhost:8080/hermes
```

Comment:
Is this used? @MormonJesus69420

Reply:
Yes, that's what validates the JWT audience now, instead of doing it by hand